src/components/CopyButton/CopyButton.tsx

```diff
@@ -6,15 +6,8 @@ import copyToClipboard from "../../utils/helpers/copyToClipboard"
 import { ButtonProps, Button } from "../Button/Button"
 import { ThemeCss } from "../../theme"
 
-const baseCss: ThemeCss = _theme => ({
-  "html:not([dir=rtl]) &": {
-    borderTopLeftRadius: 0,
-    borderBottomLeftRadius: 0,
-  },
-  "html[dir=rtl] &": {
-    borderTopRightRadius: 0,
-    borderBottomRightRadius: 0,
-  },
+const baseCss: ThemeCss = theme => ({
+  fontWeight: _theme.fontWeights.semiBold,
```

Comment:
```suggestion
  fontWeight: theme.fontWeights.semiBold,
```

`_theme` is undefined:

<img width="1268" alt="image" src="https://user-images.githubusercontent.com/21834/168882535-fa765f47-d3e8-4126-841f-d97514178f7f.png">